functions.go

```diff
@@ -172,21 +172,27 @@ Crypto Functions:
 	- genPrivateKey: Generate a private key for the given cryptosystem. If no
 	  argument is supplied, by default it will generate a private key using
 	  the RSA algorithm. Accepted values are `rsa`, `dsa`, and `ecdsa`.
-
+	- derivePassword: Derive a password from the given parameters according to the ["Master Password" algorithm](http://masterpasswordapp.com/algorithm.html)
```

Comment:
The markdown on this line won't get resolved by the godoc tool. Can you just remove the square brackets?